server/api/Routers/OrderRouter.js

```diff
@@ -0,0 +1,34 @@
+const router = require('express').Router();
+const { Order, User } = require('../../db/models/');
+
+// Routes:
+// /api/orders (all orders)
+router.get('/orders', async (req, res) => {
+  const orders = await Order.findAll();
+  res.send(orders);
```

Comment:
should we use res.json() for everything or send? @dr3mit @naimoon6450

Reply:
res.json has a tiny bit more functionality... not only that but it also converts null/undefined/primaries to json so lets stick with res.json

Reply:
also if we're using async/await we should probably do some try/catching for some kind of error logging

Reply:
I'll add try catches around the async, I completely forgot that was a thing.
Also will change the res.send()s to res.jsons().

Reply:
Yeah same here haha then I looked at the code and was like... but how do you catch errors 😅